app/swagger/requests/gibct/institutions.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+module Swagger
+  module Requests
+    module Gibct
+      class Institutions
+        include Swagger::Blocks
+
+        swagger_path '/v0/gi/institutions/autocomplete' do
+          operation :get do
+            key :description, 'Retrieves all institution names begining with a set of letters'
+            key :operationId, 'gibctInstitutionsAutocomplete'
+            key :tags, %w(institutions autocomplete)
+
+            parameter name: :term, in: :query,
+                      required: true, type: :string, description: 'start of an institution name'
+
+            response 200 do
+              key :description, 'autocomplete response'
+
+              schema do
+                key :'$ref', :GibctInstitutionsAutocomplete
+              end
+            end
+          end
+        end
+
+        swagger_path '/v0/gi/institutions/search' do
+          operation :get do
+            key :description, 'Retrieves institutions with a partial match for names, or match of city or facility code'
+            key :operationId, 'gibctInstitutionsSearch'
+            key :tags, %w(institutions search)
+
+            parameter name: :term, in: :query,
+                      required: false, type: :string, description: '(partial) institution name, city, or facility code'
+
+            response 200 do
+              key :description, 'search response'
+
+              schema do
+                key :'$ref', :GibctInstitutionsSearch
+              end
+            end
+          end
+        end
+
+        swagger_path '/v0/gi/institutions/{id}' do
+          operation :get do
+            key :description, 'Get details about an institution'
+            key :operationId, 'showInstitution'
+            key :tags, %w(institutions show)
+
+            parameter name: :id, in: :path, required: true, type: :integer,
+                      description: 'facility code of the institution'
+
+            response 200 do
+              key :description, 'show response'
+
+              schema do
+                key :'$ref', :GibctInstitution
+              end
+            end
+
+            response 400 do
```

Comment:
@patrickvinograd how was this resolved? In the middleware that gibct and rx/sm both share there is this:
```
def raise_error!
            if status&.between?(400, 599)
              raise Common::Exceptions::BackendServiceException.new(service_i18n_key, response_values, @status, @body)
            else
              raise BackendUnhandledException, "Unhandled Exception - status: #{@status}, body: #{@body}"
            end
          end
```